src/Rector/AbstractRector/BetterStandardPrinterTrait.php

```diff
@@ -78,4 +79,30 @@ protected function areNodesEqual($firstNode, $secondNode): bool
     {
         return $this->betterStandardPrinter->areNodesEqual($firstNode, $secondNode);
     }
+
+    /**
+     * Check if node found in previous node
+     */
+    protected function isNodeFoundInPrevious(Node $node, Node $find): bool
+    {
+        $previous = $node->getAttribute(AttributeKey::PREVIOUS_NODE);
+        while ($previous) {
+            $isFound = (bool) $this->betterNodeFinder->findFirst($previous, function (Node $n) use ($find): bool {
+                return $this->betterStandardPrinter->areNodesEqual($n, $find);
+            });
+
+            if ($isFound) {
+                return true;
+            }
+
+            $previous = $previous->getAttribute(AttributeKey::PREVIOUS_NODE);
+        }
+
+        $parent = $node->getAttribute(AttributeKey::PARENT_NODE);
+        if (! $parent instanceof Node) {
+            return false;
+        }
+
+        return $this->isNodeFoundInPrevious($parent, $find);
+    }
```

Comment:
Thai should be part of not finde. Printer should only přitom.

Reply:
implemented.